js/script.js

```diff
@@ -1,50 +1,79 @@
-let bookLib = JSON.parse(localStorage.getItem('bookLib')) || [];
-function getBook() {
-  const book = {};
-  book.title = document.querySelector('.title').value;
-  book.author = document.querySelector('.author').value;
-  book.id = '';
-
-  if (bookLib === null) {
-    book.id = 0;
-  } else {
-    book.id = bookLib.length;
+class Book {
+  id = 0;
+
+  title = '';
+
+  author = '';
+
+  constructor(id, title, author) {
+    this.id = id;
+    this.author = author;
+    this.title = title;
+  }
+}
+
+class bookLib {
+  bookList = [];
+
+  constructor(bookList) {
+    this.bookList = bookList;
+  }
+
+  addBook(book) {
+    this.bookList.push(book);
+    localStorage.setItem('bookLib', JSON.stringify(this.bookList));
+    UI(book);
   }
+
+  removeBook(book) {
+    const newBooks = [];
+    for (let i = 0; i < this.bookList.length; i++) {
+      if (this.bookList[i].id !== book.id) {
+        newBooks.push(this.bookList[i]);
+      }
```

Comment:
- [ ] - [ ] [OPTIONAL] Here also you can just filter the book using array.filter() method , You should try to use the array methods that do not mutate the original array,  favor the use of methods  e.g. [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) instead of [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push), [pop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop), [splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) and so on. (see code example)